sig-documentation/README.md

```diff
@@ -1,5 +1,11 @@
 # Spinnaker Documentation Special Interest Group
 
+## Meeting Info
+* First Wednesday of every month at 11:00 AM Pacific/2:00 PM Eastern
+* [Agenda and notes](https://www.google.com/url?q=https://docs.google.com/document/d/1uErNfK3Yr6LKELNSO7YfsRFyJVHqjivfzIO4nBmMCUw/edit?usp%3Dsharing&sa=D&ust=1571845713765000&usg=AOvVaw0Tv3BNYsj99yoV4B1kGHB-)
```

Comment:
This is a redirect link, copied from a chat or something? I'd replace it with this:
https://docs.google.com/document/d/1uErNfK3Yr6LKELNSO7YfsRFyJVHqjivfzIO4nBmMCUw/edit